src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,73 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String[][] salaryByName = new String[names.length][2];
```

Comment:
Don't use two-dimension arrays as it's mentioned in the checklist. Instead, consider creating a class to represent the employee's salary details or use a different approach to store this information.